### PR TITLE
Project Guess Who Ulysses 

### PR DESCRIPTION
**Pull request by Ulve17**

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -253,33 +254,48 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
+  board.style.display = 'flex';
+  winOrLose.style.display = 'none';
   // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
```
Comment (Ingetora) [inline code/script.js]: It seams like you get an error in the console when the page is loaded and no question is selected yet.

Uncaught TypeError: Cannot read property 'value' of undefined
    at HTMLButtonElement.checkQuestion (script.js:307)

File: code/script.js [hunk 2]
```diff
@@ -341,3 +357,4 @@ start()
 
 // All the event listeners
 restartButton.addEventListener('click', start)
+questions.addEventListener('change', () => selectQuestion())
```
Comment (Ingetora) [inline code/script.js:403]: Very clean code and easy to read!

File: code/script.js [hunk 3]
```diff
@@ -271,81 +275,125 @@ const selectQuestion = () => {
...
+  } else if (category === 'eye color') {
+    if (keep) {
+      alert(
+        `Yeeep, the person has ${value} eyes! Keeping all the people with ${value} eyes.`
+      )
+    } else {
+      alert(
+        `Negative!, the person doesn't have ${value} eyes. Removing people with ${value} eyes.`
+      )
+    }
```
Comment (Ingetora) [inline code/script.js]: Love your alert messages!

File: code/script.js [hunk 4]
```diff
@@ -288,54 +287,51 @@ const selectQuestion = () => {
...
 }
 
 // This function should be invoked when you click on 'Find Out'.
 const checkQuestion = () => {
-  const keep = currentQuestion.value === secret[currentQuestion.attribute]
-  filterCharacters(keep)
-  // Compare the currentQuestion with the secret person.
+    const keep = currentQuestion.value === secret[currentQuestion.attribute]
+    filterCharacters(keep)
+   // Compare the currentQuestion with the secret person.
```
Comment (Ingetora) [inline code/script.js:309]: Really nice code!

File: README.md [hunk 5]
```diff
@@ -1,13 +1,12 @@
 # Project Name
 
-Replace this readme with your own information about your project.
+This is a version of Guess Who game. The user must identify a person based on three guesses. Then the user must make a choice. We are using javascript conditionals , arrys and objects.
```
Comment (Ingetora) [inline README.md:4]: I think I can make more than three guesses. But that was not in the scope anyhow!

**Review by Ingetora — COMMENTED** [review]
Really nice game! I like the styling and your code is very clean and easy to read. Indentations by the book. Great work this week. 👍 //Maria

File: code/script.js [hunk 6]
```diff
@@ -262,82 +270,136 @@ const selectQuestion = () => {
...
 const guess = (suspect) => {
-  // store the interaction from the player in a variable.
+  const confirmed = confirm(`Are you really sure you want to guess on ${suspect}? Maybe you have got it wrong.`)
+
+  if (confirmed) {
+    checkMyGuess(suspect)
+  }
+// store the interaction from the player in a variable.
   // remember the confirm() ?
   // If the player wants to guess, invoke the checkMyGuess function.
```
Comment (KarolCislowski) [inline code/script.js:379]: remember to always clean your code from all console.log()s and commented out code blocks or guides from the teachers 🤓

File: code/script.js [hunk 7]
```diff
@@ -253,7 +257,11 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
+  board.style.display = 'flex'
+  winOrLose.style.display = 'none'
```
Comment (KarolCislowski) [inline code/script.js:261]: nice way to hide and show your elements 👍🏻

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

I know it was an extra point of this assignment to style it your way but remember that there is no another way to master CSS then practice it over and over again 🦾 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**